src/frontend/js/utils/OrderHelper/index.ts

```diff
@@ -95,6 +96,8 @@ export class OrderHelper {
   }
 
   static getFailedInstallment(order: Order) {
-    return order.payment_schedule?.find((installment) => installment.state === 'refused');
+    return order.payment_schedule?.find(
+      (installment) => installment.state === PaymentScheduleState.FAILED,
```

Comment:
The state defined in the backend are this one : 

```
PAYMENT_STATE_PENDING = "pending"
PAYMENT_STATE_PAID = "paid"
PAYMENT_STATE_REFUSED = "refused"
```

Failed is not a valid one.

Reply:
Oh yes good catch !